### ISSUE TITLE
[V2] Stand up staging environment

### ISSUE THREAD

**Issue by bora-skylight** [CDCgov/ReportVision#242]
Acceptance Criteria

- ~~Provision a resource group for an environment called `Staging` using Terraform~~
- ~~Deploy frontend & OCR API to it~~
- ~~Ensure frontend is publicly accessible with URL~~
- Add capability in github actions to automatically deploy `Main` branch to this environment once a new commit is merged

What we need to do before being able to deploy to a staging environment:
- ~~Ensure TLS is enabled properly~~
- ~~Setup process to auto-wipe demo environments on a cron, timer or multiple times a day.~~
- Banner for non-production saying something like: "DO NOT USE PERSONAL INFORMATION! THIS IS NOT A PRODUCTION ENVIRONMENT!"

Stretch goals:
- Make it so networks are already provisioned before Terraform is applied. This will better prepare us if and/or when we request for a CDC owned Azure account. Currently we are creating our networks every time we create a new environment and that is not how the CDC operates in the cloud.
